scripts/tests/test_validate_docstrings.py

```diff
@@ -1004,6 +1006,32 @@ def test_item_subsection(self, idx, subsection):
         assert result[idx][3] == subsection
 
 
+class TestDocstringClass(object):
+    @pytest.mark.parametrize('name_and_expected_obj',
+                             [('pandas.isnull', pandas.isnull),
+                              ('pandas.DataFrame', pandas.DataFrame),
+                              ('pandas.Series.sum', pandas.Series.sum)])
+    def test_resolves_class_name(self, name_and_expected_obj):
+        name, expected_obj = name_and_expected_obj
+        d = validate_docstrings.Docstring(name)
+        assert d.obj is expected_obj
+
+    @pytest.mark.parametrize('invalid_name', ['panda', 'panda.DataFrame'])
+    def test_raises_for_invalid_module_name(self, invalid_name):
+        # When an invalid module name is supplied, an ImportError should be
+        # raised, and the message should contain the user-supplied name.
+        msg = 'No module can be imported.*{}'.format(invalid_name)
+        with pytest.raises(ImportError, match=msg):
```

Comment:
Just a general comment but if we ever think about dropping 3.5 support this is probably better suited as a `ModuleNotFoundError` (no change needed for now though)